workshop/Lab1/README.md

```diff
@@ -33,7 +33,7 @@ that has already been built and uploaded to DockerHub under the name
    This action will take a bit of time. To check the status of the running application,
```

Comment:
Feels like "a bit of time" should be clarified unless this is something that devs will understand to mean a few minutes? hours?